android/quest/src/main/res/drawable/background_warning.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/background_warning" />
+    <corners android:radius="@dimen/item_margin_vertical" />
+
+
+</shape>
```

Comment:
I do not think this is used anywhere in compose code so it can be removed. You can apply the background using compose APIs